src/PKSim.Infrastructure/Services/DefaultIndividualRetriever.cs

```diff
@@ -49,6 +50,7 @@ public Individual DefaultIndividualFor(SpeciesPopulation speciesPopulation)
             var individualDTO = _individualDefaultValueRetriever.DefaultSettingFor(speciesPopulation);
```

Comment:
This code is all a bit backward here. We are creating a dto to create an origin data which in turnes create an individual. 
I think this should all be based on origindata as we do for Matlab etc.. Not relevant for now but should be changed